### ISSUE TITLE
SQS Reporter V2 SDK

### ISSUE THREAD

**Issue by llinder** [openzipkin/zipkin-aws#140]
When projects are already importing the V2 AWS SDK for other things, it would be nice to have the option of using V2 SDK for the SQS reporter as well.  Besides not inflating the project dependencies, reportedly important V1 and V2 in the same project produces conflicts.

**Comment by adriancole** [comment]
totally agree. I was thinking the same thing. Also the collectors have no
reason to be v1 as we have no reason to re-use classes from them or retain
compatibility.

**Comment by devinsba** [comment]
+1 the collectors should really be updated to V2 since the server is now
netty based, we can remove that unneeded dependency of another http library

**Comment by llinder** [comment]
Updating the collector makes sense.  I will open a new issue and MR once the sender side of things is done.
